presto-kafka07/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -76,7 +76,17 @@
     /**
      * Fetch size
      */
-    private int fetchSize = 10 * 1024 * 1024;
+    private DataSize fetchSize = new DataSize(10, Unit.MEGABYTE);
+
+    /**
+     * Default Query Interval
```

Comment:
This is a variable. So:
1. Don't call it `defaultXXXX`.
2. Default value should be a `final` value. Here, it is good to have one final Duration value called `DEFAULT_QUERY_RANGE` and a variable called `queryInterval`. And then assign the default value of the variable with the constant value.

Also, can you elaborate more about what this variable means? I have to read through the logic code using this variable to understand it.